Size of long may be 32-bit and thus can't hold MS_YEAR.

--- a/deps/ms.c	2013-12-01 18:34:08.000000000 +0100
+++ b/deps/ms.c	2015-09-09 22:47:16.476393763 +0200
@@ -91,7 +91,7 @@
 milliseconds_to_string(long long ms) {
   char *str = malloc(MS_MAX);
   if (!str) return NULL;
-  long div = 1;
+  long long div = 1;
   char *fmt;
 
   if (ms < MS_SEC) fmt = "%lldms";
@@ -113,7 +113,7 @@
 
 char *
 milliseconds_to_long_string(long long ms) {
-  long div;
+  long long div;
   char *name;
 
   char *str = malloc(MS_MAX);
